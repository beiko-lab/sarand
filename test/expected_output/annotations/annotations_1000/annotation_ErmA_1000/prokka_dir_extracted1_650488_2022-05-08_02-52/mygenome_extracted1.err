Discrepancy Report Results

Summary
FATAL: MISSING_PROTEIN_ID:2 proteins have invalid IDs.
DISC_SOURCE_QUALS_ASNDISC:strain (all present, all unique)
DISC_SOURCE_QUALS_ASNDISC:taxname (all present, all unique)
DISC_FEATURE_COUNT:CDS: 2 present
DISC_COUNT_NUCLEOTIDES:1 nucleotide Bioseqs are present
FEATURE_LOCATION_CONFLICT:2 features have inconsistent gene locations.
DISC_QUALITY_SCORES:Quality scores are missing on all sequences.
ONCALLER_COMMENT_PRESENT:1 comment descriptors were found (all same)
MISSING_GENOMEASSEMBLY_COMMENTS:1 bioseqs are missing GenomeAssembly structured comments
MOLTYPE_NOT_MRNA:1 molecule types are not set as mRNA.
TECHNIQUE_NOT_TSA:1 technique are not set as TSA
MISSING_STRUCTURED_COMMENT:1 sequences do not include structured comments.
MISSING_PROJECT:3 sequences do not include project.
DISC_INCONSISTENT_MOLINFO_TECH:Molinfo Technique Report (some missing, all same)


Detailed Report

FATAL: DiscRep_ALL:MISSING_PROTEIN_ID::2 proteins have invalid IDs.
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence:_1 (length 243)
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence:_2 (length 260)

DiscRep_ALL:DISC_SOURCE_QUALS_ASNDISC::strain (all present, all unique)
DiscRep_SUB:DISC_SOURCE_QUALS_ASNDISC::1 sources have unique values for strain
DiscRep_ALL:DISC_SOURCE_QUALS_ASNDISC::taxname (all present, all unique)
DiscRep_SUB:DISC_SOURCE_QUALS_ASNDISC::1 sources have unique values for taxname
DiscRep_ALL:DISC_FEATURE_COUNT::CDS: 2 present
DiscRep_ALL:DISC_COUNT_NUCLEOTIDES::1 nucleotide Bioseqs are present
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2732)

DiscRep_ALL:FEATURE_LOCATION_CONFLICT::2 features have inconsistent gene locations.
DiscRep_SUB:FEATURE_LOCATION_CONFLICT::Coding region xref gene does not exist
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:CDS	rRNA adenine N-6-methyltransferase	sequence::1001-1732	ECADEPCO_00001

DiscRep_SUB:FEATURE_LOCATION_CONFLICT::Coding region xref gene does not exist
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:CDS	Streptomycin 3''-adenylyltransferase	sequence::c2640-1858	ECADEPCO_00002

DiscRep_ALL:DISC_QUALITY_SCORES::Quality scores are missing on all sequences.

DiscRep_ALL:ONCALLER_COMMENT_PRESENT::1 comment descriptors were found (all same)
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence::Annotated using prokka 1.14.5 from https://github.com/tseemann/prokka

DiscRep_ALL:MISSING_GENOMEASSEMBLY_COMMENTS::1 bioseqs are missing GenomeAssembly structured comments
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2732)

DiscRep_ALL:MOLTYPE_NOT_MRNA::1 molecule types are not set as mRNA.
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2732)

DiscRep_ALL:TECHNIQUE_NOT_TSA::1 technique are not set as TSA
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2732)

DiscRep_ALL:MISSING_STRUCTURED_COMMENT::1 sequences do not include structured comments.
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2732)

DiscRep_ALL:MISSING_PROJECT::3 sequences do not include project.
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2732)
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence:_1 (length 243)
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence:_2 (length 260)

DiscRep_ALL:DISC_INCONSISTENT_MOLINFO_TECH::Molinfo Technique Report (some missing, all same)
DiscRep_SUB:DISC_INCONSISTENT_MOLINFO_TECH::technique (all missing)
DiscRep_SUB:DISC_INCONSISTENT_MOLINFO_TECH::1 Molinfos are missing field technique
prokka_dir_extracted1_650488_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2732)

