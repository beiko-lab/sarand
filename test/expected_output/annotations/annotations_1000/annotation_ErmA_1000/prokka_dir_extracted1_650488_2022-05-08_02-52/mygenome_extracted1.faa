>ECADEPCO_00001 rRNA adenine N-6-methyltransferase
MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFTKELVKMSRSVTAIEI
DGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPYNISTDIVKRITFESQA
KYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYFHPKPSVDSVLIVLERHQ
PLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVTNINKLSKEQFLSIFNSYK
LFH
>ECADEPCO_00002 Streptomycin 3''-adenylyltransferase
MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRINSDVDVLVVVNHSLPQ
LTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPKREFIYGEWLRGEFENG
QIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIRRAIKDSLPELIEGIKGD
ERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLLDIARKGYRGECDDKWEGL
YSKVKALVKYMKNSIETSLN
