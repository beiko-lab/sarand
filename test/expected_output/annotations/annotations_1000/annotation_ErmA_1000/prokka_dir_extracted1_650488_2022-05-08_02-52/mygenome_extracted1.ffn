>ECADEPCO_00001 rRNA adenine N-6-methyltransferase
ATGAACCAGAAAAACCCTAAAGACACGCAAAATTTTATTACTTCTAAAAAGCATGTAAAA
GAAATATTGAATCACACGAATATCAGTAAACAAGACAACGTAATAGAAATCGGATCAGGA
AAAGGACATTTTACCAAAGAGCTAGTCAAAATGAGTCGATCAGTTACTGCTATAGAAATT
GATGGAGGCTTATGTCAAGTGACTAAAGAAGCGGTAAACCCCTCTGAGAATATAAAAGTG
ATTCAAACGGATATTCTAAAATTTTCCTTCCCAAAACATATAAACTATAAGATATATGGT
AATATTCCTTATAACATCAGTACGGATATTGTCAAAAGAATTACCTTTGAAAGTCAGGCT
AAATATAGCTATCTTATCGTTGAGAAGGGATTTGCGAAAAGATTGCAAAATCTGCAACGA
GCTTTGGGTTTACTATTAATGGTGGAGATGGATATAAAAATGCTCAAAAAAGTACCACCA
CTATATTTTCATCCTAAGCCAAGTGTAGACTCTGTATTGATTGTTCTTGAACGACATCAA
CCATTGATTTCAAAGAAGGACTACAAAAAGTATCGATCTTTTGTTTATAAGTGGGTAAAC
CGTGAATATCGTGTTCTTTTCACTAAAAACCAATTCCGACAGGCTTTGAAGCATGCAAAT
GTCACTAATATTAATAAACTATCGAAGGAACAATTTCTTTCTATTTTCAATAGTTACAAA
TTGTTTCACTAA
>ECADEPCO_00002 Streptomycin 3''-adenylyltransferase
ATGAGCAATTTGATTAACGGAAAAATACCAAATCAAGCGATTCAAACATTAAAAATCGTA
AAAGATTTATTTGGAAGTTCAATAGTTGGAGTATATCTATTTGGTTCAGCAGTAAATGGT
GGTTTACGCATTAACAGCGATGTAGATGTTCTAGTCGTCGTGAATCATAGTTTACCTCAA
TTAACTCGAAAAAAACTAACAGAAAGACTAATGACTATATCAGGAAAGATTGGAAATACG
GATTCTGTTAGACCACTTGAAGTTACGGTTATAAATAGGAGTGAAGTTGTCCCTTGGCAA
TATCCTCCAAAAAGAGAATTTATATACGGTGAGTGGCTCAGGGGTGAATTTGAGAATGGA
CAAATTCAGGAACCAAGCTATGATCCTGATTTGGCTATTGTTTTAGCACAAGCAAGAAAG
AATAGTATTTCTCTATTTGGTCCTGATTCTTCAAGTATACTTGTCTCCGTACCTTTGACA
GATATTCGAAGAGCAATTAAGGATTCTTTGCCAGAACTAATTGAGGGGATAAAAGGTGAT
GAGCGTAATGTAATTTTAACCCTAGCTCGAATGTGGCAAACAGTGACTACTGGTGAAATT
ACCTCGAAAGATGTCGCTGCAGAATGGGCTATACCTCTTTTACCTAAAGAGCATGTAACT
TTACTGGATATAGCTAGAAAAGGCTATCGGGGAGAGTGTGATGATAAGTGGGAAGGACTA
TATTCAAAGGTGAAAGCACTCGTTAAGTATATGAAAAATTCTATAGAAACTTCTCTCAAT
TAG
