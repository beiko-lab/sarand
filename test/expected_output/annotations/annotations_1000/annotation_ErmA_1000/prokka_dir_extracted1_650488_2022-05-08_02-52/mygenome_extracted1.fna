>sequence:
AAAAACTCTTGGATTGCCCCTAATTTATAAACATAGGTAGGCGGTGTCTCGACTTCAGAT
ACATTATCAAGAATAACTATTCTTCCTTCCTCATTCAACAGTTCCTTCATTTGCTGTATT
ACGCTGGCTATATCATCCAAATGATGAAAGGTTGTTCGGCTTACAATAAAATCAAACTTC
TCATTAAAATTAAGTTGTTCTGCATTCATATTCAGATAGACCGTATTTGTTAGTTGACGT
TTAGATTTGGCAAGATCGAGCATTTGATTAGAAATATCAATCCCTACCACTTCATCATAA
TAACTTGCTAATTTCTCCACTAACAAACCCGAGCCACATCCGATATCTAATGCTCTGCCT
TTCTTTGGAGACATATTAGACACAAAGAATGAATAATCATTCAAAAGCTCATTCACGAAA
TCGTAATCTTCTGCAACCTTATCAAACTGTGATTCTATTGTATTCAAAAAGATCCCCCAT
TCCTACTTTATCGACATTCTTTCATTACTTACCACTTTAGATGTTTTTTCGTTGGGGATA
AAACTTCCCTTTAGACAATTTTATCCAAAGACAATACAACAGTGCAACTTTATTAAAGTC
ACTGTCCTTTATCGCAGCCTTTACTTTTTAGTAAAGACAGTGGCTTCTCTTATCAAGTTT
CAAAACATATTATTTTGAAGAAAACGTCCATCTGAAGTGTCAAGTGCAAAATTACATATA
AAGGTTTATTCTAAAATGAAAAGATGATACAATCATATTCAGTTACATAAGGAGGTTTCA
ATTATGTGCACCAGTATCGCAGTAGTAGAAATTACTTTATCTCATTCATAATGAAAAAAA
TGGAAAGGAGATAAAAGTATGGGTACTTTTTCTATATTTGTTATTAATAAAGTTCGTTAT
CAACCAAATCAAAATTAATTGGTTATAATGAACGCTTAATGTCAGTTCATTATAACCAGT
AAGGAGAAGGTTATAATGAACCAGTAAGGAGAAGGTTATAATGAACCAGAAAAACCCTAA
AGACACGCAAAATTTTATTACTTCTAAAAAGCATGTAAAAGAAATATTGAATCACACGAA
TATCAGTAAACAAGACAACGTAATAGAAATCGGATCAGGAAAAGGACATTTTACCAAAGA
GCTAGTCAAAATGAGTCGATCAGTTACTGCTATAGAAATTGATGGAGGCTTATGTCAAGT
GACTAAAGAAGCGGTAAACCCCTCTGAGAATATAAAAGTGATTCAAACGGATATTCTAAA
ATTTTCCTTCCCAAAACATATAAACTATAAGATATATGGTAATATTCCTTATAACATCAG
TACGGATATTGTCAAAAGAATTACCTTTGAAAGTCAGGCTAAATATAGCTATCTTATCGT
TGAGAAGGGATTTGCGAAAAGATTGCAAAATCTGCAACGAGCTTTGGGTTTACTATTAAT
GGTGGAGATGGATATAAAAATGCTCAAAAAAGTACCACCACTATATTTTCATCCTAAGCC
AAGTGTAGACTCTGTATTGATTGTTCTTGAACGACATCAACCATTGATTTCAAAGAAGGA
CTACAAAAAGTATCGATCTTTTGTTTATAAGTGGGTAAACCGTGAATATCGTGTTCTTTT
CACTAAAAACCAATTCCGACAGGCTTTGAAGCATGCAAATGTCACTAATATTAATAAACT
ATCGAAGGAACAATTTCTTTCTATTTTCAATAGTTACAAATTGTTTCACTAAATTAAAGT
AATAAAGCGTTCTCTAATTTCACAAGAGGACGCTTTATTCTTCCCAAAAATTGTTCAATA
TTTATCAATAAATCAGTAGTTTTAAAAGTAAGCACCTGTTATTGCAATAAAATTAGCCTA
ATTGAGAGAAGTTTCTATAGAATTTTTCATATACTTAACGAGTGCTTTCACCTTTGAATA
TAGTCCTTCCCACTTATCATCACACTCTCCCCGATAGCCTTTTCTAGCTATATCCAGTAA
AGTTACATGCTCTTTAGGTAAAAGAGGTATAGCCCATTCTGCAGCGACATCTTTCGAGGT
AATTTCACCAGTAGTCACTGTTTGCCACATTCGAGCTAGGGTTAAAATTACATTACGCTC
ATCACCTTTTATCCCCTCAATTAGTTCTGGCAAAGAATCCTTAATTGCTCTTCGAATATC
TGTCAAAGGTACGGAGACAAGTATACTTGAAGAATCAGGACCAAATAGAGAAATACTATT
CTTTCTTGCTTGTGCTAAAACAATAGCCAAATCAGGATCATAGCTTGGTTCCTGAATTTG
TCCATTCTCAAATTCACCCCTGAGCCACTCACCGTATATAAATTCTCTTTTTGGAGGATA
TTGCCAAGGGACAACTTCACTCCTATTTATAACCGTAACTTCAAGTGGTCTAACAGAATC
CGTATTTCCAATCTTTCCTGATATAGTCATTAGTCTTTCTGTTAGTTTTTTTCGAGTTAA
TTGAGGTAAACTATGATTCACGACGACTAGAACATCTACATCGCTGTTAATGCGTAAACC
ACCATTTACTGCTGAACCAAATAGATATACTCCAACTATTGAACTTCCAAATAAATCTTT
TACGATTTTTAATGTTTGAATCGCTTGATTTGGTATTTTTCCGTTAATCAAATTGCTCAT
GATTTCACCTCGTTGATTATGTTCATATAAAGTTTATATTGATACTCAATTTACTTACCC
TAGATTGGACATATACTTAAATTACTGTTCAA
