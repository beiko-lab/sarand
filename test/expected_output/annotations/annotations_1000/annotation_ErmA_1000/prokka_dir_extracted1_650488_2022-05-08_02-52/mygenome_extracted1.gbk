LOCUS       sequence:               2732 bp    DNA     linear       08-MAY-2022
DEFINITION  Genus species strain strain.
ACCESSION   
VERSION
KEYWORDS    .
SOURCE      Genus species
  ORGANISM  Genus species
            Unclassified.
COMMENT     Annotated using prokka 1.14.5 from
            https://github.com/tseemann/prokka.
FEATURES             Location/Qualifiers
     source          1..2732
                     /organism="Genus species"
                     /mol_type="genomic DNA"
                     /strain="strain"
     CDS             1001..1732
                     /gene="ermA1"
                     /locus_tag="ECADEPCO_00001"
                     /EC_number="2.1.1.184"
                     /inference="ab initio prediction:Prodigal:002006"
                     /inference="similar to AA sequence:UniProtKB:P0A0H2"
                     /codon_start=1
                     /transl_table=11
                     /product="rRNA adenine N-6-methyltransferase"
                     /translation="MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHF
                     TKELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNI
                     PYNISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPP
                     LYFHPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKH
                     ANVTNINKLSKEQFLSIFNSYKLFH"
     CDS             complement(1858..2640)
                     /gene="ant1"
                     /locus_tag="ECADEPCO_00002"
                     /EC_number="2.7.7.47"
                     /inference="ab initio prediction:Prodigal:002006"
                     /inference="similar to AA sequence:UniProtKB:P0A0D1"
                     /codon_start=1
                     /transl_table=11
                     /product="Streptomycin 3''-adenylyltransferase"
                     /translation="MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRI
                     NSDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYP
                     PKREFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLT
                     DIRRAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEH
                     VTLLDIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN"
ORIGIN      
        1 aaaaactctt ggattgcccc taatttataa acataggtag gcggtgtctc gacttcagat
       61 acattatcaa gaataactat tcttccttcc tcattcaaca gttccttcat ttgctgtatt
      121 acgctggcta tatcatccaa atgatgaaag gttgttcggc ttacaataaa atcaaacttc
      181 tcattaaaat taagttgttc tgcattcata ttcagataga ccgtatttgt tagttgacgt
      241 ttagatttgg caagatcgag catttgatta gaaatatcaa tccctaccac ttcatcataa
      301 taacttgcta atttctccac taacaaaccc gagccacatc cgatatctaa tgctctgcct
      361 ttctttggag acatattaga cacaaagaat gaataatcat tcaaaagctc attcacgaaa
      421 tcgtaatctt ctgcaacctt atcaaactgt gattctattg tattcaaaaa gatcccccat
      481 tcctacttta tcgacattct ttcattactt accactttag atgttttttc gttggggata
      541 aaacttccct ttagacaatt ttatccaaag acaatacaac agtgcaactt tattaaagtc
      601 actgtccttt atcgcagcct ttacttttta gtaaagacag tggcttctct tatcaagttt
      661 caaaacatat tattttgaag aaaacgtcca tctgaagtgt caagtgcaaa attacatata
      721 aaggtttatt ctaaaatgaa aagatgatac aatcatattc agttacataa ggaggtttca
      781 attatgtgca ccagtatcgc agtagtagaa attactttat ctcattcata atgaaaaaaa
      841 tggaaaggag ataaaagtat gggtactttt tctatatttg ttattaataa agttcgttat
      901 caaccaaatc aaaattaatt ggttataatg aacgcttaat gtcagttcat tataaccagt
      961 aaggagaagg ttataatgaa ccagtaagga gaaggttata atgaaccaga aaaaccctaa
     1021 agacacgcaa aattttatta cttctaaaaa gcatgtaaaa gaaatattga atcacacgaa
     1081 tatcagtaaa caagacaacg taatagaaat cggatcagga aaaggacatt ttaccaaaga
     1141 gctagtcaaa atgagtcgat cagttactgc tatagaaatt gatggaggct tatgtcaagt
     1201 gactaaagaa gcggtaaacc cctctgagaa tataaaagtg attcaaacgg atattctaaa
     1261 attttccttc ccaaaacata taaactataa gatatatggt aatattcctt ataacatcag
     1321 tacggatatt gtcaaaagaa ttacctttga aagtcaggct aaatatagct atcttatcgt
     1381 tgagaaggga tttgcgaaaa gattgcaaaa tctgcaacga gctttgggtt tactattaat
     1441 ggtggagatg gatataaaaa tgctcaaaaa agtaccacca ctatattttc atcctaagcc
     1501 aagtgtagac tctgtattga ttgttcttga acgacatcaa ccattgattt caaagaagga
     1561 ctacaaaaag tatcgatctt ttgtttataa gtgggtaaac cgtgaatatc gtgttctttt
     1621 cactaaaaac caattccgac aggctttgaa gcatgcaaat gtcactaata ttaataaact
     1681 atcgaaggaa caatttcttt ctattttcaa tagttacaaa ttgtttcact aaattaaagt
     1741 aataaagcgt tctctaattt cacaagagga cgctttattc ttcccaaaaa ttgttcaata
     1801 tttatcaata aatcagtagt tttaaaagta agcacctgtt attgcaataa aattagccta
     1861 attgagagaa gtttctatag aatttttcat atacttaacg agtgctttca cctttgaata
     1921 tagtccttcc cacttatcat cacactctcc ccgatagcct tttctagcta tatccagtaa
     1981 agttacatgc tctttaggta aaagaggtat agcccattct gcagcgacat ctttcgaggt
     2041 aatttcacca gtagtcactg tttgccacat tcgagctagg gttaaaatta cattacgctc
     2101 atcacctttt atcccctcaa ttagttctgg caaagaatcc ttaattgctc ttcgaatatc
     2161 tgtcaaaggt acggagacaa gtatacttga agaatcagga ccaaatagag aaatactatt
     2221 ctttcttgct tgtgctaaaa caatagccaa atcaggatca tagcttggtt cctgaatttg
     2281 tccattctca aattcacccc tgagccactc accgtatata aattctcttt ttggaggata
     2341 ttgccaaggg acaacttcac tcctatttat aaccgtaact tcaagtggtc taacagaatc
     2401 cgtatttcca atctttcctg atatagtcat tagtctttct gttagttttt ttcgagttaa
     2461 ttgaggtaaa ctatgattca cgacgactag aacatctaca tcgctgttaa tgcgtaaacc
     2521 accatttact gctgaaccaa atagatatac tccaactatt gaacttccaa ataaatcttt
     2581 tacgattttt aatgtttgaa tcgcttgatt tggtattttt ccgttaatca aattgctcat
     2641 gatttcacct cgttgattat gttcatataa agtttatatt gatactcaat ttacttaccc
     2701 tagattggac atatacttaa attactgttc aa
//
