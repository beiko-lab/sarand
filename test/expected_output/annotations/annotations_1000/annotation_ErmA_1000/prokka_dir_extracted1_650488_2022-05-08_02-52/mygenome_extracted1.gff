##gff-version 3
##sequence-region sequence: 1 2732
sequence:	Prodigal:002006	CDS	1001	1732	.	+	0	ID=ECADEPCO_00001;eC_number=2.1.1.184;Name=ermA1;gene=ermA1;inference=ab initio prediction:Prodigal:002006,similar to AA sequence:UniProtKB:P0A0H2;locus_tag=ECADEPCO_00001;product=rRNA adenine N-6-methyltransferase
sequence:	Prodigal:002006	CDS	1858	2640	.	-	0	ID=ECADEPCO_00002;eC_number=2.7.7.47;Name=ant1;gene=ant1;inference=ab initio prediction:Prodigal:002006,similar to AA sequence:UniProtKB:P0A0D1;locus_tag=ECADEPCO_00002;product=Streptomycin 3''-adenylyltransferase
##FASTA
>sequence:
AAAAACTCTTGGATTGCCCCTAATTTATAAACATAGGTAGGCGGTGTCTCGACTTCAGAT
ACATTATCAAGAATAACTATTCTTCCTTCCTCATTCAACAGTTCCTTCATTTGCTGTATT
ACGCTGGCTATATCATCCAAATGATGAAAGGTTGTTCGGCTTACAATAAAATCAAACTTC
TCATTAAAATTAAGTTGTTCTGCATTCATATTCAGATAGACCGTATTTGTTAGTTGACGT
TTAGATTTGGCAAGATCGAGCATTTGATTAGAAATATCAATCCCTACCACTTCATCATAA
TAACTTGCTAATTTCTCCACTAACAAACCCGAGCCACATCCGATATCTAATGCTCTGCCT
TTCTTTGGAGACATATTAGACACAAAGAATGAATAATCATTCAAAAGCTCATTCACGAAA
TCGTAATCTTCTGCAACCTTATCAAACTGTGATTCTATTGTATTCAAAAAGATCCCCCAT
TCCTACTTTATCGACATTCTTTCATTACTTACCACTTTAGATGTTTTTTCGTTGGGGATA
AAACTTCCCTTTAGACAATTTTATCCAAAGACAATACAACAGTGCAACTTTATTAAAGTC
ACTGTCCTTTATCGCAGCCTTTACTTTTTAGTAAAGACAGTGGCTTCTCTTATCAAGTTT
CAAAACATATTATTTTGAAGAAAACGTCCATCTGAAGTGTCAAGTGCAAAATTACATATA
AAGGTTTATTCTAAAATGAAAAGATGATACAATCATATTCAGTTACATAAGGAGGTTTCA
ATTATGTGCACCAGTATCGCAGTAGTAGAAATTACTTTATCTCATTCATAATGAAAAAAA
TGGAAAGGAGATAAAAGTATGGGTACTTTTTCTATATTTGTTATTAATAAAGTTCGTTAT
CAACCAAATCAAAATTAATTGGTTATAATGAACGCTTAATGTCAGTTCATTATAACCAGT
AAGGAGAAGGTTATAATGAACCAGTAAGGAGAAGGTTATAATGAACCAGAAAAACCCTAA
AGACACGCAAAATTTTATTACTTCTAAAAAGCATGTAAAAGAAATATTGAATCACACGAA
TATCAGTAAACAAGACAACGTAATAGAAATCGGATCAGGAAAAGGACATTTTACCAAAGA
GCTAGTCAAAATGAGTCGATCAGTTACTGCTATAGAAATTGATGGAGGCTTATGTCAAGT
GACTAAAGAAGCGGTAAACCCCTCTGAGAATATAAAAGTGATTCAAACGGATATTCTAAA
ATTTTCCTTCCCAAAACATATAAACTATAAGATATATGGTAATATTCCTTATAACATCAG
TACGGATATTGTCAAAAGAATTACCTTTGAAAGTCAGGCTAAATATAGCTATCTTATCGT
TGAGAAGGGATTTGCGAAAAGATTGCAAAATCTGCAACGAGCTTTGGGTTTACTATTAAT
GGTGGAGATGGATATAAAAATGCTCAAAAAAGTACCACCACTATATTTTCATCCTAAGCC
AAGTGTAGACTCTGTATTGATTGTTCTTGAACGACATCAACCATTGATTTCAAAGAAGGA
CTACAAAAAGTATCGATCTTTTGTTTATAAGTGGGTAAACCGTGAATATCGTGTTCTTTT
CACTAAAAACCAATTCCGACAGGCTTTGAAGCATGCAAATGTCACTAATATTAATAAACT
ATCGAAGGAACAATTTCTTTCTATTTTCAATAGTTACAAATTGTTTCACTAAATTAAAGT
AATAAAGCGTTCTCTAATTTCACAAGAGGACGCTTTATTCTTCCCAAAAATTGTTCAATA
TTTATCAATAAATCAGTAGTTTTAAAAGTAAGCACCTGTTATTGCAATAAAATTAGCCTA
ATTGAGAGAAGTTTCTATAGAATTTTTCATATACTTAACGAGTGCTTTCACCTTTGAATA
TAGTCCTTCCCACTTATCATCACACTCTCCCCGATAGCCTTTTCTAGCTATATCCAGTAA
AGTTACATGCTCTTTAGGTAAAAGAGGTATAGCCCATTCTGCAGCGACATCTTTCGAGGT
AATTTCACCAGTAGTCACTGTTTGCCACATTCGAGCTAGGGTTAAAATTACATTACGCTC
ATCACCTTTTATCCCCTCAATTAGTTCTGGCAAAGAATCCTTAATTGCTCTTCGAATATC
TGTCAAAGGTACGGAGACAAGTATACTTGAAGAATCAGGACCAAATAGAGAAATACTATT
CTTTCTTGCTTGTGCTAAAACAATAGCCAAATCAGGATCATAGCTTGGTTCCTGAATTTG
TCCATTCTCAAATTCACCCCTGAGCCACTCACCGTATATAAATTCTCTTTTTGGAGGATA
TTGCCAAGGGACAACTTCACTCCTATTTATAACCGTAACTTCAAGTGGTCTAACAGAATC
CGTATTTCCAATCTTTCCTGATATAGTCATTAGTCTTTCTGTTAGTTTTTTTCGAGTTAA
TTGAGGTAAACTATGATTCACGACGACTAGAACATCTACATCGCTGTTAATGCGTAAACC
ACCATTTACTGCTGAACCAAATAGATATACTCCAACTATTGAACTTCCAAATAAATCTTT
TACGATTTTTAATGTTTGAATCGCTTGATTTGGTATTTTTCCGTTAATCAAATTGCTCAT
GATTTCACCTCGTTGATTATGTTCATATAAAGTTTATATTGATACTCAATTTACTTACCC
TAGATTGGACATATACTTAAATTACTGTTCAA
