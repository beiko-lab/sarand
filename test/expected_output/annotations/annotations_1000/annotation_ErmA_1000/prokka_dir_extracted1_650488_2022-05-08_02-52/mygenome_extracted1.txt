organism: Genus species strain 
contigs: 1
bases: 2732
CDS: 2
